site/styles/main.css

```
@@ -0,0 +1,148 @@
+* {
+    box-sizing: border-box;
+  }
+
+  body {
+    background-color: #E7E6DB;  
+    color: #3A3B47;             
+    font-family: Arial, sans-serif;
+  }
+
+  .width-wrap {
+    background-color: #A3A56E; 
+    max-width: 960px;
+    margin: 0 auto 3em;
+    padding: 0 2em 2em; 
+    border: 1px solid rgba(0, 0, 0, 0.3);
+    border-top: none;
+    position: relative;
+  }
+
+  footer {
+    width: clamp(300px, 50vw, 890px);
+    margin: 3em auto 2em;                
+    border: none;                        
+    border-bottom: 3px dotted #2e4834;   
+  }
+
+  dt::after {
+    content: ":";
+  }
+
+  header {
+    border-top: 4px solid rgba(0, 0, 0, 0.1);
+    margin-left: -2em; 
+    margin-right: -2em; 
+    padding-left: 2em;
+    padding-right: 2em;
+  }
+
+  header nav {
+    border-bottom: 1px dotted rgba(0, 0, 0, 0.3);
+    padding-left: 0;  
+    padding-right: 0;
+  }
```

> 9. `Lab 5: Styling > site header > nav > left/right padding matches width-wrap`

This is almost right, you just have the values in the wrong rule.
```suggestion
/*
 * I used shorthand but you don't need to!
 */
  header {
    border-top: 4px solid rgba(0, 0, 0, 0.1);
    margin: 0 -2em; 
  }

  header nav {
    border-bottom: 1px dotted rgba(0, 0, 0, 0.3);
    padding: 0 2em;  
  }
```